ui/lib/objects.ts

```diff
@@ -199,3 +206,15 @@ export class Provider extends FluxObject {
     return this.obj.spec.channel || "";
   }
 }
+
+export class Alert extends FluxObject {
+  get providerRef(): string {
+    return this.obj.spec.providerRef.name || "";
```

Comment:
Shouldn't this have a namespace as well? 

Most refs in flux (see: sourceRef) have an optional namespace, where no value means "same as current" so to make references correct I keep finding myself having to add if statements to discover the correct namespace - I would expect to see something similar here.

Reply:
The docs led me here as the type for `providerRef` - is this the same thing like the uid where every object actually has one?
https://pkg.go.dev/github.com/fluxcd/pkg/apis/meta?utm_source=godoc#LocalObjectReference

Reply:
Oh, you're right, it's always local. Nevermind, then.

Still valid for event sources, though.